docs/gettingstarted/installation.rst

```diff
@@ -263,19 +263,18 @@ You'll probably want to setup at least two nodes.
    At the time of writing, the only supported Fedora 20 kernel on DigitalOcean is ``Fedora 20 x64 vmlinuz-3.16.6-203.fc20.x86_64``.
```

Comment:
Yeah, well since our automated tests now select what ever is currently the newest kernel, I've updated the documentation along the lines of  "choose the newest kernel".